### PR TITLE
(#321) Update Create Custom Package Templates page

### LINKED ISSUE #321: Update documentation for template command
With the introduction of the `template` command, the documentation needs to be updated:

* Update the documentation at https://docs.chocolatey.org/en-us/guides/create/create-custom-package-templates

### PR DESCRIPTION
**Pull request by pauby**
## Description Of Changes
Update Create Custom Packages templates page to reflect the new `choco template` command.

## Motivation and Context
The `choco template` command changes the way we work with custom templates.

## Testing
Used the Docker image to view the changes.

## Change Types Made

* [ ] Bug fix (non-breaking change)
* [ ] Feature / Enhancement (non-breaking change)
* [ ] Breaking change (fix or feature that could cause existing functionality to change)
* [ ] PowerShell code changes.

## Related Issue

Fixes #321

## Change Checklist

* [x] Requires a change to the documentation
* [x] Documentation has been updated
* [ ] Tests to cover my changes, have been added
* [ ] All new and existing tests passed.
* [ ] PowerShell v2 compatibility checked.

### CODE REVIEW

**Comment by pauby** [comment]
This should not be merged until Chocolatey CLI 0.12.0 is released.

**Comment by gep13** [comment]
@pauby that is a good catch!

**Comment by pauby** [comment]
This is now ready for review.

File: input/en-us/guides/create/create-custom-package-templates.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ You would call `choco new pkgname -t organization` and choco will use the templa
 
 ### Can I replace the built-in template?
 
-To replace the built-in template, you should put a folder in the template with the name of "default". Then choco will use that instead of the built-in template with no need to specify a template name.
+If you are using Chocolatey CLI version 0.12.0 or later, you can use the [`choco template` command](https://docs.chocolatey.org/en-us/create/commands/template) to list the installed templates and [set the `defaultTemplateName`](xref:configuration#general) to set the template to be used by default when running the `choco new` command.
+
+If you are using a version of Chocolatey CLI earlier than 0.12.0 then you can replace the built-in template by copying the template to a folder called `default`. The `choco new` command will then use that by default.
```
Comment (TheCakeIsNaOH) [inline input/en-us/guides/create/create-custom-package-templates.md:27]: `defaultTemplateName` is now the recommended way to set the default template, but v0.12.0 still has the code for using the manually copied `default` template that overrides the built-in template. 

This sentence could be read as implying that the manually copied `default` template would no longer work in v0.12.0.

File: input/en-us/guides/create/create-custom-package-templates.md [hunk 2]
```diff
@@ -103,24 +105,20 @@ $packageArgs = @{
...
+If you have Chocolatey v0.9.10+, you can manage templates as packages allowing you to upgrade a template when a new version is available. When it comes to packaging templates, Chocolatey takes a conventional approach. You must create a package with the suffix ".template" and have a templates folder.
 
-To manage a template as a package, create a new package with the name "templatename.template". The name of the package minus the ".template" will be the name of the template.
+To manage a template as a package, create a new package with the name "<TEMPLATE NAME>.template". The name of the package minus the ".template" will be the name of the template.
 
-Then create a templates folder. This is where the template goes. the only thing to remember is that the nuspec file created here must end in ".template" as a Chocolatey package allows only one nuspec file.
+Then create a templates folder in the default Chocolatey install location. This is where the template goes. The only thing to remember is that the nuspec file created here must end in ".template" as a Chocolatey package allows only one nuspec file.
```
Comment (TheCakeIsNaOH) [inline input/en-us/guides/create/create-custom-package-templates.md:116]: The templates folder in the `.template` package should be next to the `<template name>.template.nuspec`, not in `$env:ChocolateyInstall` 

```suggestion
Then create a templates folder next to the `<TEMPLATE NAME>.template.nuspec`. This folder is where the template goes. The only thing to remember is that the nuspec file created here must end in ".template" (e.g. `<TEMPLATE NAME>.nuspec.template`) as a Chocolatey package allows only one nuspec file.
```

File: input/en-us/guides/create/create-custom-package-templates.md (hunk 1 above)
Comment (pauby) [reply input/en-us/guides/create/create-custom-package-templates.md:27]: I've amended that to be clearer.

File: input/en-us/guides/create/create-custom-package-templates.md (hunk 2 above)
Comment (pauby) [reply input/en-us/guides/create/create-custom-package-templates.md:116]: Done. I was looking at the File Explorer screenshot above when writing this!

**Review by TheCakeIsNaOH — DISMISSED** [review]

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@pauby your changes have been merged, thanks for your contribution 👍

**Pull request merged**